src/libraries/System.Runtime/tests/System.IO.FileSystem.Tests/FileSystemTest.cs

```diff
@@ -11,11 +11,11 @@ public abstract partial class FileSystemTest : FileCleanupTestBase
     {
         public static readonly byte[] TestBuffer = { 0xBA, 0x5E, 0xBA, 0x11, 0xF0, 0x07, 0xBA, 0x11 };
 
-        public static bool AreAllLongPathsAvailable => PathFeatures.AreAllLongPathsAvailable();
+        public static bool AreAllLongPathsAvailable => true;
```

Comment:
Delete `AreAllLongPathsAvailable`, `LongPathsAreNotBlocked` and `UsingNewNormalization` since they are always true